learning/views.py

```diff
@@ -53,9 +66,17 @@ def learning_process(request):
     delta_time = timezone.now() - request.session['last_record_time']
     request.session['last_record_time'] = timezone.now()
     request.user.last_study_duration += delta_time
+    request.user.last_study_vocab_count += 1
```

Comment:
this view is the general view for learning, which means it will also handle review and relearn. Therefore `last_study_vocab_count` shouldn't be updated here...